store/src/java/com/zimbra/cs/account/ldap/Validators.java

```diff
@@ -67,7 +67,7 @@ private synchronized long getNextCheck() {
 
         @Override
         public void validate(Provisioning prov, String action, Object... args) throws ServiceException {
-            if (args.length < 1) return;
+            if (null == args || args.length < 1) return;
```

Comment:
I don't think its possible for varargs to be null. Did you run into an NPE?

Reply:
This code has now been removed due to changes in logic.